kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/AlCustomizationSettings.java

```diff
@@ -12,9 +12,9 @@
 public class AlCustomizationSettings extends JsonMarker {
 
     private String customMessageBackgroundColor = "#e6e5ec";
-    private String sentMessageBackgroundColor = "#5c5aa7";
+    private String sentMessageBackgroundColor = "";
```

Comment:
worst case if server API fails, anything will break on design?

Reply:
Nothing will break as the fallback color is from the colors.xml file which will always be there.
The sequence is:
```
   if(color is set in applozic-settings) {
      use this color
   } else if(color received from server){
      use server color
} else {
    use the default color from colors.xml file
}
```